DEV-GUIDE.md

```diff
@@ -80,6 +89,11 @@ export GOPATH=~/go
 export PATH=$PATH:$GOPATH/bin
 export ISTIO=$GOPATH/src/istio.io # eg. ~/go/src/istio.io
 
+# Please change HUB and TAG to a desired HUB and TAG for custom docker
+# container builds.
+export HUB="gcr.io/yourid"
```

Comment:
probably should be docker.io ? (not everyone is on gcr.io)

Reply:
sounds good.